app/services/invitation_manager.rb

```diff
@@ -28,7 +29,7 @@ def success?
     def process_emails
       emails.reduce([]) do |bad_emails, email|
         invitation = @user.invitations.new(email: email, status: 0, role: @role)
-        invitation.save ? invitation.send! : bad_emails << email
+        invitation.save ? invitation.send!(@url) : bad_emails << email
```

Comment:
I did consider this, but that means we need to make a column on the invitations table for url, and I don't think that's necessary. Admittedly, without it we need to pass the url along a long connect-the-dots pattern to get it from the controller to the method that actually needs it. If there's a way to accomplish this without saving it to the database that I'm not considering, let me know.